db.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from sqlalchemy import create_engine, MetaData, Table, Column, Integer, String
+
+SQLITE_DB_PATH = 'styles.db'
+SQLITE_URL = f'sqlite:///{SQLITE_DB_PATH}'
```

Comment:
Better use combined `DATABASE_URL` and make it possible to override from `os.environ` (https://12factor.net/backing-services).